src/test/java/org/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -186,6 +292,17 @@ void testTitleChecks() {
         assertCorrect(withMode(createContext(StandardField.TITLE, "This is a {Title}"), BibDatabaseMode.BIBTEX));
         assertCorrect(withMode(createContext(StandardField.TITLE, "{C}urrent {C}hronicle"), BibDatabaseMode.BIBTEX));
         assertCorrect(withMode(createContext(StandardField.TITLE, "{A Model-Driven Approach for Monitoring {ebBP} BusinessTransactions}"), BibDatabaseMode.BIBTEX));
+        assertCorrect(withMode(createContext(StandardField.TITLE, "This is a sub title 1: This is a sub title 2"), BibDatabaseMode.BIBTEX));
```

Comment:
I would move all these tests to `TitleCheckerTest` and just keep one subtitle tests here.

This test class here is only for an integration test (TitleChecker is integrated in the IntegrityCheck) - the "real" functionality is in `org.jabref.logic.integrity.TitleChecker`.

When you are on it, you could fix the capitalization in TitleCheckerTest:

![grafik](https://user-images.githubusercontent.com/1366654/78498685-072f9480-774c-11ea-82e2-ad0bc6e7caea.png)